docs/content/ActionList.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: ActionList
+---
+
+An `ActionList` is a list of items which can be activated or selected. `ActionList` is the base component for many of our menu-type components, including `DropdownMenu` and `ActionMenu`.
+
+## Default example
+
+```javascript live noinline
+const Demo = () => {
+  return (
+    <ActionList
+      items={[
+        {text: 'New file'},
+        ActionList.Divider,
+        {text: 'Copy link'},
+        {text: 'Edit file'},
+        {text: 'Delete file', variant: 'danger'}
+      ]}
+    />
+  )
+}
+
+render(<Demo />)
+```
+
```

Comment:
A complex example of using groups could be a nice addition to the docs.

Reply:
Added in https://github.com/primer/components/pull/1078/commits/04c0648edc6a83fc56e5028bdf9ebb6e3fa4a797!